aliases/available/msys2.aliases.bash

```diff
@@ -13,3 +13,4 @@ test -n "$LS_COMMON" && alias ls="command ls $LS_COMMON"
 alias ll="ls -l"
 alias la="ls -a"
 alias lal="ll -a"
+alias ls="ls -F"
```

Comment:
This overwrites the `ls` alias settings above (see line 12).

If used, the `-F` should be made part of `LS_COMMON`